src/map/entities/petentity.cpp

```diff
@@ -286,8 +278,12 @@ void CPetEntity::loadPetZoningInfo()
         health.tp = static_cast<uint16>(master->petZoningInfo.petTP);
         health.hp = master->petZoningInfo.petHP;
         health.mp = master->petZoningInfo.petMP;
-        setJugDuration(master->petZoningInfo.jugDuration);
-        setJugSpawnTime(master->petZoningInfo.jugSpawnTime);
+
+        if (m_PetType == PET_TYPE::JUG_PET)
```

Comment:
nit: this check is not necessary as `petZoningInfo.jugDuration` will just contain default values for non jug pets

Reply:
I put that check there because of debug breaks throwing `[critical]` errors to console if the pet wasn't a jug pet.